### ISSUE TITLE
Adding the graphical components to the resulted.php file.

### ISSUE THREAD

**Issue by c20adael** [HGustavs/LenaSYS#11379]
This is reffering to the issue #11107

**Comment by c20adael** [comment]
How is this coming along? @a20kimar

**Comment by a20kimar** [comment]
> How is this coming along? @a20kimar

It's going well. I just saw the message in the issue about functionality regarding resulted not being used much. I've just finished the layout and so on. The current version of lenasys has a semi-hardcoded way of handling the different checkboxes but I think I got around that fairly nicely. I'll commit and push so someone can have a look at whether this is worth continuing on or not.

**Comment by a20kimar** [comment]
My solution should now be available for anyone in the issue branch. The previous method was understandably coded for one solution of one window of checkboxes and I had to make it viable for two or more. I added php elements to represent the new button and checkbox window. I then had to implement so that it functions independently from the other one, it's own css id's, classes and names and then also change or add to the javascript where needed to have this function on its own. For example would the "select all" button work on both of them at the same time while it should only work on the elements in that window. I also made sure that the elements are dynamically changed depending on the table header which was the case for the dugga filter as well.

For a more in-depth view check out the commit containing all the changes on the branch for this issue.

This is how the finished product look.

<img width="383" alt="Screen Shot 2022-04-08 at 14 34 19" src="https://user-images.githubusercontent.com/82010032/162436536-f88b18e6-0e22-48ab-a4ca-a9255021770d.png">
